radicle-node/src/service.rs

```diff
@@ -739,7 +750,7 @@ where
 
     /// Handle an announcement message.
     ///
-    /// Returns `true` if this announcement should be stored and relayed to connected peers,
+    /// Returns `true` if this announcement should be relayed to connected peers,
```

Comment:
Announcements are stored and redelivered to connecting nodes to sync them up if they've been offline :)